client/src/pages/Profile.js

```diff
@@ -17,6 +18,9 @@ export default function Profile({ authProps, setAuthProps }) {
   const [selectedTab, setSelectedTab] = React.useState(TABS[0]);
   const [editingInfo, setEditingInfo] = React.useState(false);
 
+  const [dietitian, setDietitian] = React.useState("");
```

Comment:
If the [dietitian, setDietitian] state is being used as a boolean type, please adjust the initial state to be that of boolean instead of string.

Reply:
It is supposed to be strings, for one part I was just testing something and used boolean, but it is deleted now